src/plugins/instrument_document_load.js

```diff
@@ -22,24 +24,33 @@ class InstrumentPageLoad {
             return;
         }
 
+        this._ensureSpanStarted(tracerImp);
```

Comment:
This is the crux of the change: more the span creation so it happens as soon as possible (rather than in the `readystatechange` handler).